### ISSUE TITLE
Virtuel scrolling in kirby-list

### ISSUE THREAD

**Issue by jvh2** [kirbydesign/designsystem#853]
Issue:
The postinglist which uses kirbylist - load-on-demand is performing badly when dealing with many transactions or loading transaction in from the top of the list (expand upcoming). The bad performance is mainly due that everything in the list is re-rendered upon changes in the list.

Solution:
Instead of rendering the entire list at a time, only a subset of the items that fits on the screen (plus a small buffer) will be rendered. As the user navigates, a new subset of items is calculated and rendered, re-using the existing DOM if desired. From (https://web.dev/virtualize-lists-with-angular-cdk/)

Links:
https://www.thecodecampus.de/blog/virtual-scrolling-in-angular-7/
https://web.dev/virtualize-lists-with-angular-cdk/
https://material.angular.io/cdk/scrolling/overview

**Comment by jakobe** [comment]
https://ionicframework.com/docs/api/virtual-scroll

**Comment by lydemann** [comment]
På baggrund af vores snak blev vi enige om:
- Lave virtual scroll inde i kirby-list (som separat presentation component)
- Slå virtual scroll til via en enableVirtualScroll property
- Vi dokumenterer at der ikke kan bruges list header og section header samtidig i en virtual scroll list

**Comment by jakobe** [comment]
Also see: https://github.com/ionic-team/ionic-framework/issues/22460
TLDR; ion-virtual-scroll will be deprecated, use `cdkScrollable` from [Angular Material CDK](https://material.angular.io/cdk/scrolling/overview)

**Comment by flemcito** [comment]
A bit of research: https://github.com/dhilt/vscroll
This lib has both an angular wrapper and a native JS - which is a huge win as we talk web components. Need to test it though
